benchmarks/domains/synthetic_2C1D_1C.py

```diff
@@ -10,10 +10,11 @@
 
 from baybe.campaign import Campaign
 from baybe.parameters import NumericalContinuousParameter, NumericalDiscreteParameter
-from baybe.recommenders.pure.nonpredictive.sampling import RandomRecommender
+from baybe.recommenders import RandomRecommender
```

Comment:
wasnt it our convention that internal code always imports from the specific source and not the high level import?

Reply:
Yes, for baybe code. We haven't agreed on a policy for the benchmark code though, which is a form of consuming code. I can live with both, I only made it consistent within the file

Reply:
makes sense